bibliopixel/drivers/serial/driver.py

```diff
@@ -144,7 +146,14 @@ def _compute_packet(self):
         self._packet.extend([0] * self._bufPad)
 
     def _send_sync(self):
-        self._com.write(self._sync_packet)
+        self._serial('write', self._sync_packet)
+
+    def _serial(self, method, *args):
```

Comment:
This is going to slow things down because it's got to do these weird abstractions every time. We need serial to be as fast as possible.